tool/gravity/cli/plan.go

```diff
@@ -64,9 +64,15 @@ func initUpdateOperationPlan(localEnv, updateEnv *localenv.LocalEnvironment) err
 	return trace.Wrap(err)
 }
 
-func displayOperationPlan(localEnv, updateEnv, joinEnv *localenv.LocalEnvironment, operationID string, format constants.Format) error {
-	operations, err := getLastOperation(localEnv, updateEnv, joinEnv, operationID)
+func displayOperationPlan(localEnv *localenv.LocalEnvironment, environ LocalEnvironmentFactory, operationID string, format constants.Format) error {
+	operations, err := getLastOperation(localEnv, environ, operationID)
 	if err != nil {
+		if trace.IsNotFound(err) {
+			return trace.NotFound(`no operation found.
+This usually means that the installation/join operation has failed to start or was not started.
+Clean up the node with 'gravity leave' if necessary and start the installation/join with either 'gravity install' or 'gravity join'.
```

Comment:
How is a user supposed to know if running gravity leave is necessary?

Reply:
I'll have the code check and output the appropriate message.